inference/core/workflows/core_steps/utils/dominant_color.py

```diff
@@ -0,0 +1,79 @@
+from abc import ABC, abstractmethod
```

Comment:
suggestion: would be good to provide some rough estimation of how fast it is in docs - for our and clients' awareness

Reply:
So I tested this on a few different images. 

The rough estimation is between 0.0925 seconds and 0.4992 seconds
The two drivers of speed are the "Color Complexity" and the Image Size. 
It appears from testing that how complex the colors are drives times more though. 

Where do you suggstion I provide this information? Should this go in the long description of the json schema or somewhere else?

Reply:
let's stick to short description for now